utils/messages.ts

```diff
@@ -0,0 +1,51 @@
+import { CreatedMessageWithoutContent } from "@pagopa/io-functions-commons/dist/generated/definitions/CreatedMessageWithoutContent";
+import { MessageModel } from "@pagopa/io-functions-commons/dist/src/models/message";
+import { ServiceModel } from "@pagopa/io-functions-commons/dist/src/models/service";
+import { BlobService } from "azure-storage";
+import * as A from "fp-ts/lib/Apply";
+import * as E from "fp-ts/lib/Either";
+import { flow, pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import * as TE from "fp-ts/lib/TaskEither";
+
+/**
+ * This function enrich a CreatedMessageWithoutContent with
+ * service's details and message's subject.
+ *
+ * @param messageModel
+ * @param serviceModel
+ * @param blobService
+ * @returns
+ */
+export const enrichMessagesData = (
+  messageModel: MessageModel,
+  serviceModel: ServiceModel,
+  blobService: BlobService
+) => (messages: readonly CreatedMessageWithoutContent[]) =>
```

Comment:
Can we be explicit on resul type?

```
) => (messages: readonly CreatedMessageWithoutContent[]): XXX =>
```

Reply:
Fixe in https://github.com/pagopa/io-functions-app/pull/203/commits/1cf1b28e1b0a804d1843054480a4029665ad1e53